nixos/modules/services/desktops/pipewire/pipewire.nix

```diff
@@ -116,6 +116,16 @@ in {
         };
       };
 
+      audio = {
+        enable = lib.mkOption {
+          type = lib.types.bool;
+          # this is for backwards compatibility
```

Comment:
Generally, it looks good to me. But I am still not sure about this option (since backwards compatibility carries the connotation of old vs. new to me). What do you expect the path forward to be?

- The option will remain alongside with `{pulse,alsa,jack}.enable` forever. Then it is redundant (except for disabling this when using PulseAudio as the main soundserver and JACK-on-PW? Is that really a thing?)
- The options will remain but will need to be manually activated.
- The individual options will be removed. Then we lose granularity.

Reply:
The path forward is to eventually default this to false, because there will be more PW-native audio-only clients (think EasyEffects), and you'd want to explicitly opt into using PW for audio.